samplePDF/samplePDFFDBase.cpp

```diff
@@ -1414,11 +1358,103 @@ void samplePDFFDBase::addData(TH2D* Data) {
   }
 }
 
+void samplePDFFDBase::SetupNuOscillator() {
+  OscProbCalcerFactory* OscProbCalcFactory = new OscProbCalcerFactory();
```

Comment:
OscProbCalcFactory  is not used fron what I see

Reply:
No that is correct, this can be removed (It was from previous implementation where I was just directly dealing with OscProbCalcerBase objects)